reconstruct/reconstruct_test.go

```diff
@@ -16,20 +15,19 @@ var input = []struct {
 	{"testdata/hackett_newick", 66}}
 
 func Test_reconstruct(t *testing.T) {
-	start := time.Now()
 	for _, test := range input {
 
-		//example of how to run simulation: 1) read in tree (no fastas) 2) simulate random gene 3) simulate evolution 4) remove ancestors for reconstruction 4) set up fastas
-		tre, er := tree_newick.ReadNewick(test.newick_filename)
+		//example of how to run simulation: 1) read in tree (no fastas) 2) simulate random gene 3) simulate evolution 4) remove ancestors for reconstruction 4) assign fastas
+		tre, er := expandedTree.ReadNewick(test.newick_filename)
 		if er != nil {
 		}
-		simulate.Rand_gene("test", test.length)
+		simulate.RandGene("test", test.length, 0.42) //galGal6
```

Comment:
I would create a variable that is called gcContent and set it to 0.42, then pass in gcContent to the function.  This makes it easier to read for someone seeing the code for the first time.  Otherwise they might not be sure what the 0.42 is for.